### PR TITLE
Buildpack + Manifest-less Deploy Support

### PR DESCRIPTION
**Pull request by WillPlatnick**

### CODE REVIEW

**Comment by tcg** [comment]
:eyes:

Excited about what I see in docs. I'm not going to claim I properly reviewed all the scripts, or fully grasp what's happening, but didn't see anything obvs out of place to me.

File: deploy [hunk 1]
```diff
@@ -66,6 +65,30 @@ get_secrets_for_usage
...
+      cp /opt/kubernetes-deploy/manifests/other/autoscale.yaml $CI_PROJECT_DIR/kubernetes/production/autoscale-$line.yaml
+      sed -i -e "s/{{PREFIX}}/$line/g" $CI_PROJECT_DIR/kubernetes/deployment-$line.yaml
+      sed -i -e "s/{{PREFIX}}/$line/g" $CI_PROJECT_DIR/kubernetes/production/autoscale-$line.yaml
+      set_prefix_defaults $line
+    done
+  fi
+fi
+
+echo "Setting defaults"
+set_defaults
```
Comment (bman7777) [inline deploy]: setting defaults two times?  I think I see why you need the 2nd one, but I don't understand the 1st one.

Comment (bman7777) [reply deploy]: I was thinking you would want to set_defaults AFTER set_prefix_defaults (which the IF statement does.)

Comment (bman7777) [reply deploy]: actually on second thought...one doesn't affect the other.  You're right...

**Pull request merged**